src/nft/components/details/detailsV2/LandingPage.tsx

```diff
@@ -0,0 +1,100 @@
+import Column from 'components/Column'
+import Row from 'components/Row'
+import { VerifiedIcon } from 'nft/components/icons'
+import { CollectionInfoForAsset, GenieAsset } from 'nft/types'
+import styled from 'styled-components/macro'
+import { BREAKPOINTS, ThemedText } from 'theme'
+
+const MAX_WIDTH = 560
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  min-height: ${({ theme }) => `calc(100vh - ${theme.navHeight}px)`};
+  align-items: center;
+  padding: 22px 20px 0px;
+  gap: 26px;
+
+  @media screen and (min-width: ${BREAKPOINTS.sm}px) {
+    gap: 64px;
+    padding-top: 28px;
+  }
+
+  @media screen and (min-width: ${BREAKPOINTS.xl}px) {
+    flex-direction: row;
+    padding-top: 0px;
+    padding-bottom: ${({ theme }) => `${theme.navHeight}px`};
+    gap: 80px;
+  }
+`
+
+const InfoContainer = styled(Column)`
+  width: 100%;
+  align-items: center;
+  gap: 40px;
+
+  @media screen and (min-width: ${BREAKPOINTS.sm}px) {
+    width: ${MAX_WIDTH}px;
+  }
+`
+
+const InfoDetailsContainer = styled(Column)`
+  gap: 4px;
+  align-items: center;
+`
+
+const MediaContainer = styled.div`
+  width: 100%;
+  height: 100%;
+
+  @media screen and (min-width: ${BREAKPOINTS.sm}px) {
+    width: ${MAX_WIDTH}px;
+    width: ${MAX_WIDTH}px;
+  }
+`
+
+const StyledMedia = styled.img`
+  object-fit: contain;
+  height: 100%;
+  width: 100%;
+`
+
+const StyledSubheaderText = styled(ThemedText.SubHeaderSmall)`
+  line-height: 20px;
+
+  @media screen and (min-width: ${BREAKPOINTS.sm}px) {
+    line-height: 24px !important;
+    font-size: 16px !important;
+  }
+`
+
+const StyledHeadlineText = styled(ThemedText.HeadlineSmall)`
+  @media screen and (min-width: ${BREAKPOINTS.sm}px) {
+    line-height: 44px !important;
+    font-size: 36px !important;
+  }
+`
+
+interface LandingPageProps {
+  asset: GenieAsset
+  collection: CollectionInfoForAsset
+}
+
+export const LandingPage = ({ asset, collection }: LandingPageProps) => {
+  return (
+    <Container>
+      <MediaContainer>
+        <StyledMedia src={asset.imageUrl} />
+      </MediaContainer>
+      <InfoContainer>
+        <InfoDetailsContainer>
+          <Row justify="center" gap="4px" align="center">
+            <StyledSubheaderText>{asset.name ?? `${asset.collectionName} #${asset.tokenId}`}</StyledSubheaderText>
+            {collection.isVerified && <VerifiedIcon width="16px" height="16px" />}
+          </Row>
+          <StyledHeadlineText>{collection.collectionName}</StyledHeadlineText>
```

Comment:
I think you have the collection and asset name order reversed

Reply:
![Screen Shot 2023-04-27 at 09 35 59 ](https://user-images.githubusercontent.com/11512321/234930493-94ba4ad8-b86a-47df-9e94-e1fc29f0c618.png)

Reply:
It always seems like it should be the other way around to me